oci-seccomp-bpf-hook.go

```diff
@@ -33,6 +34,8 @@ type event struct {
 	ID uint32
 	// Command which makes the syscall
 	Command [16]byte
+	// Check exit
```

Comment:
Please add a more verbose comment. What's the semantics of this field? How and when is it used?